### PR TITLE
DB_Project2

### PR DESCRIPTION
**Pull request by Abdulkareem24**
DB_Project2_Extract from excel and added to the database

### CODE REVIEW

**Comment by vajrang-b** [comment]
SourceCode/2023/Oct2023/Projects/Abdul_Kareem/DB_Proj2/project.json
For the `NoFile` resource file:

- Score Criteria: Code Completion 2/2, Execution 2/2, Annotations and Comments 0.5/1
- You need to provide the Automation Hub URL in your project settings.

For the `Process.xaml` file:

- Score Criteria: Code Completion 1/2, Execution 2/2, Annotations and Comments 0/1
- Rename the activities in your file for clarity. Currently, they use default names which are not descriptive enough.
- The variable 'db_Con' does not follow the standard naming conventions.

For the `Main.xaml` file:

- Score Criteria: Code Completion 1/2, Execution 2/2, Annotations and Comments 0/1
- The variable 'ExtractDataTable' should start with 'dt_' as per the standard conventions.
- The activities need more descriptive names for easier understanding.

In conclusion, make sure to use more descriptive names for your activities and variables and check their formatting according to the standard conventions. Additionally, don't forget to provide all necessary project information in the settings.
